Scala/Scala.sublime-syntax

```diff
@@ -130,6 +131,48 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@specialized'
```

Comment:
This really shouldn't be special-cased.  First off, there's nothing particularly magical about `@specialized`, and other annotations *can* take type parameters in this manner.  Second, it is extremely common in practice to import-rename `@specialized` to `@sp` or `@spec`, for concision.  Your special-casing doesn't catch this.

I would rather simply support parenthetical parameters on annotations.  You'll have to do a bit of lookahead for it, I think, but that's not too bad.